### ISSUE TITLE
Wrong name for management endpoint in AzureEnvironment class

### ISSUE THREAD

**Issue by vhe1** [Azure/azure-libraries-for-net#101]
In AzureEnvironment the management endpoint is spelled wrong:
![grafik](https://user-images.githubusercontent.com/12475356/32985974-e99a601c-ccc6-11e7-879b-a6a8f829ef08.png)

The consequence is that in special clouds (like the german or chinese cloud) the resource management clients don't take the management url from the AzureCredentials but fall back to some global url, which, of course, is not valid for the special clouds.

Therefore, for every management client I have to set the url separately:
```
// log on
var servicecreds = SdkContext.AzureCredentialsFactory.FromServicePrincipal(clientId, clientSecret, tenantId, AzureEnvironment.AzureGermanCloud).WithDefaultSubscription(subscriptionId);

// create client:
SubscriptionClient subCli = new SubscriptionClient(servicecreds) {BaseUri = new Uri(servicecreds.Environment.ManagementEnpoint)};
```

**Comment by hovsepm** [comment]
@vhe1 looks like a duplicate of https://github.com/Azure/azure-libraries-for-net/issues/33 . This should be fixed in v1.4

**Comment by hovsepm** [comment]
@vhe1 to verify that fix - could you post here what is the typo?

**Comment by vhe1** [comment]
Well yes, the typo is the missing "d" in ManagementEn*point. Right where the asterisk here is and right where the cursor is in the image in my issue description above.
Yes, it looks like that issue #33, unless the error shows up somewhere else too.

_Also, while I'm reading #33_, for the "normal" cloud, "management.core.xxx", as in "management.core.com" does not work. The symptom is that I don't have any subscription when using the SubscriptionClient to get them. On the other hand, with "https://management.azure.com/" I do get my normal, non-German cloud subscription.

The currrently working (tried this afternoon) urls are
- https://management.azure.com/ for the non-special clouds
- https://management.microsoftazure.de/ for the german cloud.

Maybe before the changes in the management url's go into the next release, someone should try them out.

**Comment by hovsepm** [comment]
@vhe1 thanks for update. Initially I though that you found a typo inside the url (versus property name).  

Unfortunately it is too late for another fix in v1.4 since it is on the way out. We will address it in the next release.

**Comment by hovsepm** [comment]
This is fixed long time ago. Available in the latest release v.1.9.1
